src/domain/event/formSections/externalUserContact/ExternalUserContact.tsx

```diff
@@ -21,7 +21,11 @@ const ExternalUserContact: FC<ExternalUserContactProps> = ({
   const { t } = useTranslation();
 
   return (
-    <Fieldset heading={t('event.form.sections.contact')} hideLegend>
+    <Fieldset
+      heading={t('event.form.sections.contact')}
+      hideLegend
+      data-testid="fields-external-user-contact"
```

Comment:
You can remove this if you use `findByRole` instead of `findByTestId`

Reply:
https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/251/commits/13d1304823525678db1873e17506fc984531e410